src/components/contexts/error-handler.tsx

```diff
@@ -20,17 +21,19 @@ export const ErrorContext = React.createContext<ErrorContextType>({
  */
 const ErrorHandler: React.FC = ({ children }) => {
   const [ error, setError ] = React.useState<string>();
+  const [ errorMessage, setErrorMessage ] = React.useState<Response>();
 
   /**
    * Handles error message and tries to print any given error to logs
-   * Sends error message to sentry'
-   * TODO: Figure out proper way to log errors with sentry
+   * Sends error message to sentry
    *
    * @param message error message
    * @param err any error
    */
   const handleError = async (message: string, err?: any) => {
     setError(message);
+    console.log(err);
+    
```

Comment:
```suggestion
  const handleError = async (message: string) => {
    setError(message);
    
```
Could remove the log as the setError message is already used?